lib/src/components/NxFontAwesomeIcon/NxFontAwesomeIcon.tsx

```diff
@@ -7,16 +7,25 @@
 import React, { forwardRef } from 'react';
 import { FontAwesomeIcon, FontAwesomeIconProps as Props } from '@fortawesome/react-fontawesome';
 import classnames from 'classnames';
+import { useUniqueId } from '../../util/idUtil';
 
 export { Props } from '@fortawesome/react-fontawesome';
 
+// Hack to get the titleId prop working correct until
+// https://github.com/FortAwesome/react-fontawesome/pull/411
+// is merged
+(FontAwesomeIcon as any).defaultProps.titleId = null;
+
 /**
  * A wrapper component around FontAwesomeIcon that adds our nx-icon css class. Takes the same props as FontAwesomeIcon
  */
 const NxFontAwesomeIcon = forwardRef((props: Props, ref) => {
   // eslint-disable-next-line react/prop-types
-  const className = classnames(props.className, 'nx-icon');
-  return <FontAwesomeIcon forwardedRef={ref} { ...props } className={className} />;
+  const className = classnames(props.className, 'nx-icon'),
+      titleId = useUniqueId(''), // FA adds its own prefix to this, no need for us to add one too
+      otherProps = props.title ? { titleId } as Partial<Props> : undefined;
```

Comment:
Without these changes, whenever an `NxFontAwesomeIcon` is given a `title` it causes FA to generate a random id for the svg `<title>` element, which leads to hydration errors